### DISCUSSION TITLE
FileStorage discovery

### DISCUSSION THREAD

**Discussion by arnitolog** [tomkerkhove/promitor#2163] [Q&A]
Hello, 
I want to collect metrics for our FileShares, like FileCapacity and FileShareCapacityQuota.
It works well when I clearly specify storage account name in the scraper config. Like below:
```  - name: azure_storage_files_capacity
    description: "The amount of File storage used by the storage account"
    resourceType: FileStorage
    azureMetricConfiguration:
      metricName: FileCapacity
      aggregation:
        type: Average
    resources:
      - accountName: fb3f0e3189668
        resourceGroupName: aks-pvc
```
at the same time, I want to use discovery to find all such resources via tags.
I tried to use `type: StorageAccount` in discovery, but in this case, scraper throws an error that metrics were not found
Could you please advise what the proper configuration for discovery in this case?

**Comment by tomkerkhove** [comment]
Discovery for file share discovery is not supported at the moment: https://docs.promitor.io/v2.8/scraping/providers/file-storage/

It is being tracked in https://github.com/tomkerkhove/promitor/issues/1132

**Reply by tomkerkhove** [reply]
Contributions are welcome though, but this is not easy for Azure Storage

**Reply by arnitolog** [reply]
ok, I see. thanks for the clarification.
